custom/enikshay/integrations/ninetyninedots/repeaters.py

```diff
@@ -28,10 +33,16 @@ def allowed_to_forward(self, case):
             case.dynamic_case_properties().get('dots_99_registered') == 'false' or
             case.dynamic_case_properties().get('dots_99_registered') is None
         )
-
         return allowed_case_types_and_users and enabled and not_registered
 
     def allow_retries(self, response):
         if response is not None and response.status_code == 500:
             return True
         return False
+
+
+def create_case_repeat_records(sender, case, **kwargs):
+    create_repeat_records(NinetyNineDotsRegisterPatientRepeater, case)
+
+case_post_save.connect(create_case_repeat_records, CommCareCase)
+case_post_save.connect(create_case_repeat_records, CommCareCaseSQL)
```

Comment:
Also possibly only need this for SQL cases unless there are existing Couch domains that need this feature.